recipe/meta.yaml

```diff
@@ -18,12 +18,14 @@ source:
     - install_scripts.patch
 
 build:
-  number: 2
+  number: 3
 
 requirements:
   build:
     - python
+    - swig
     - setuptools
+    - posix  # [win]
```

Comment:
Not sure if we need that to run `make`. @mingwandroid is that correct? (Sorry to keep bothering you with `msys2` questions. I will learn eventually :smile:)